addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/channels.xml

```diff
@@ -170,4 +170,24 @@
 		<state min="0" max="360" step="1" readOnly="true"></state>
 	</channel-type>
 
+	<channel-type id="scene">
+		<item-type>Number</item-type>
+		<label>Scene</label>
+		<description>Scene channel</description>
+		<state readOnly="false">
+			<options>
+				<option value="0">Scene 1</option>
```

Comment:
I do like your idea of adding TOGGLE command.
Regarding adding an abstraction of scene switches, we already have a layer of abstraction to some extent in the binding. E.g. there is a command OnOff channel, while in RFXCOM command is a numeric value (byte), etc. I agree this is more architectural question, so let's see what @kaikreuzer says on this.

In the meantime I'll commit tests and sign-off my commits as required.